marconi-core/test/Marconi/Core/Spec/Experiment.hs

```diff
@@ -266,23 +266,23 @@ genChain cfg = flip evalStateT cfg $ do
   replicateM size genItem
 
 uniformRollBack :: TestPoint -> Gen TestPoint
-uniformRollBack = fmap TestPoint . choose . (,) 0 . unwrapTestPoint
+uniformRollBack = fmap TestPoint . chooseInt . (,) 0 . unwrapTestPoint
 
 genLargeChain
   :: (Arbitrary event)
   => Word
   -- ^ Rollback percentage
   -> Gen [Item event]
 genLargeChain p = do
-  let n = Test.choose (1000000, 1200000)
+  let n = Test.chooseInt (1000000, 1200000)
```

Comment:
That doesn't change anything right?

Reply:
According to the documentation, it should improve performance a bit (I was looking at the doc to see if it was inclusive bound or exclusive and found this note in the documentation).